### PR TITLE
Finish add ajax when order subject in course

### PR DESCRIPTION
**Pull request by hellovietnam93**

### CODE REVIEW

File: lib/rails_admin/order_subjects_in_course.rb [hunk 1]
```diff
@@ -34,10 +34,24 @@ class OrderSubjectsInCourse < RailsAdmin::Config::Actions::Base
 
         register_instance_option :controller do
           proc do
+            a = params[:course_subject]
             object.course_subjects.each do |course_subject|
-              course_subject.position = params["course_subject"].index(course_subject.id.to_s) + 1
+              unless course_subject.init?
+                a.unshift("#{course_subject.id}")
```
Comment (hienbx94) [inline lib/rails_admin/order_subjects_in_course.rb]: a.unshift("#{course_subject.id}") unless course_subject.init?

File: app/views/rails_admin/main/_subject_in_order.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    </div>
+    <div class="<%= course.init? ? "btn btn-xs btn-danger" : "hidden" %>">
+      <%= t "admin.actions.show_course_subject.locked" %>
+    </div>
+    <div class="<%= course.finish? ? "btn btn-xs btn-danger" : "hidden" %>">
+      <%= t "admin.actions.show_course_subject.finished" %>
+    </div>
+    <% if course.progress? %>
+      <div class="btn-actions-<%= course_subject.id %>">
+        <% if course.course_subjects.order_by_position(course_subject.position).first.nil? || course.course_subjects.order_by_position(course_subject.position).first.finish? %>
```
Comment (hienbx94) [inline app/views/rails_admin/main/_subject_in_order.html.erb]: xuong dong

**Comment by vietanhbkaaa** [comment]
OK

**Comment by hienbx94** [comment]
OK

**Comment by VuongLong** [comment]
OK

**Pull request merged**